### ISSUE TITLE
Can't create article on iPad

### ISSUE THREAD

**Issue by Dennis1993** [nextcloud/collectives#571]
**Describe the bug**
It is not possible to add a new article in a collective on iPad/iPhonw. I use iPadOS 16.3.1 with Safari. If I click on the PLUS icon nothing happens. The app is not useable here.

**To Reproduce**
1. Go to a collective
2. Click on the plus to add an article
3. nothing happen

**Expected behavior**
a new article should be created

**Screenshots**
![image](https://user-images.githubusercontent.com/1473674/219875021-cb1d9961-b8c7-4e29-83cf-4d380cad279c.png)

**Client details:**
 - OS: iPadOS 16.3.1
 - Browser: safari
 - Device: iPhone 12 mini and iPad Pro with iOS 16.3.1 and iPadOS 16.3.1

**Collectives app version:** (see Nextcloud apps page)
2.2.1

**Operating system:**
Ubuntu 20.0.4

**Web server:**
Apache 2.5.55

**Database:**
MariaDB 10.6

**PHP version:**
8.1.16

**Nextcloud version:** (see Nextcloud admin page)
25.0.3

**Comment by mejo-** [comment]
Dear @Dennis1993, thanks for your report. To get a better understanding on what goes wrong, we need some additional information.

Could you please open the developer console in Safari, hit the plus button one more time and send us a screenshot of the developer console? Ideally, an error or warning should be printed to that console that gives further information.

**Comment by Dennis1993** [comment]
Hey,
yes, but how can I do that on an iPhone/iPad?

**Comment by juliushaertl** [comment]
This requires Safari running on a separate machine https://www.browserstack.com/guide/how-to-debug-on-iphone

I just tested this on latest main branch and cannot reproduce in an iPad simulator running iOS 16.2

**Comment by mejo-** [comment]
@Dennis1993 could you try to get hold of a browser log the way described here: https://www.browserstack.com/guide/how-to-debug-on-iphone ?

Without that information, it's hard to understand what the problem is.

Also, does the three-dot-menu next to the plus button work? Or doesn't it work as well?

**Comment by mejo-** [comment]
I fear that we cannot do anything here without further debugging information. Will close the issue for now. Please don't hesitate to reopen if you have further findings/information.
